src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -1,92 +1,156 @@
+    value: 'descriptionLabel',
   },
   {
     label: 'Available Online',
-    value: 'tableFieldAvailableOnline',
+    value: 'availableOnlineLabel',
   },
 ];
 
-const SearchResults = ({ fetching, query, results }) => {
-  if (fetching) {
-    return <LoadingIndicator message="Loading search results..." />;
-  }
-
-  if (!results) {
-    return null;
-  }
-
-  if (!results.length) {
-    return (
-      <h2 className="vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal">
-        No results found for "<strong>{query}</strong>"
-      </h2>
-    );
-  }
-
-  return (
-    <>
-      <h2 className="vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal">
-        Showing results for "<strong>{query}</strong>"
-      </h2>
-
-      <SortableTable
-        className="vads-u-margin--0"
-        currentSort={{ order: 'ASC', value: 'title' }}
-        data={results}
-        fields={tableFields}
-      />
-    </>
-  );
-};
-
-SearchResults.propTypes = {
-  // From mapStateToProps.
-  fetching: PropTypes.bool.isRequired,
-  query: PropTypes.string.isRequired,
-  results: PropTypes.arrayOf(
-    PropTypes.shape({
-      id: PropTypes.string.isRequired,
-      type: PropTypes.string.isRequired,
-      attributes: PropTypes.shape({
+class SearchResults extends Component {
+  static propTypes = {
+    // From mapStateToProps.
+    fetching: PropTypes.bool.isRequired,
+    query: PropTypes.string.isRequired,
+    results: PropTypes.arrayOf(
+      PropTypes.shape({
+        // Original form data key-value pairs.
         firstIssuedOn: PropTypes.string.isRequired,
         formName: PropTypes.string.isRequired,
+        id: PropTypes.string.isRequired,
         lastRevisionOn: PropTypes.string.isRequired,
         pages: PropTypes.number.isRequired,
         sha256: PropTypes.string.isRequired,
         title: PropTypes.string.isRequired,
+        type: PropTypes.string.isRequired,
         url: PropTypes.string.isRequired,
+        // Table field labels that can be JSX.
+        idLabel: PropTypes.node.isRequired,
+        titleLabel: PropTypes.node.isRequired,
+        descriptionLabel: PropTypes.node.isRequired,
+        availableOnlineLabel: PropTypes.node.isRequired,
       }).isRequired,
-    }).isRequired,
-  ),
-};
+    ),
+    // From mapDispatchToProps.
+    updateResults: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedFieldLabel: 'idLabel',
+      selectedFieldOrder: 'ASC',
+    };
+  }
+
+  onHeaderClick = (fieldLabel, order) => {
+    const { selectedFieldLabel, selectedFieldOrder } = this.state;
+
+    // Derive the opposite sort order.
+    const oppositeFieldOrder = selectedFieldOrder === 'ASC' ? 'DESC' : 'ASC';
+
+    // Sort the results.
+    this.sortResults(
+      fieldLabel,
+      fieldLabel === selectedFieldLabel ? oppositeFieldOrder : order,
+    );
+  };
+
+  sortResults = (fieldLabel, selectedFieldOrder = 'ASC') => {
```

Comment:
Not sure how you feel, but a constant at the top of the file `ASCENDING` and `DESCENDING` might be nice.